test/src/main/java/org/apache/accumulo/test/functional/ScannerIT.java

```diff
@@ -113,4 +117,82 @@ public void testScannerReadaheadConfiguration() throws Exception {
       }
     }
   }
+
+  @Test
+  public void testSessionCleanup() throws Exception {
+    String tableName = getUniqueNames(1)[0];
+    final String table = getUniqueNames(1)[0];
```

Comment:
duplicate variable? Can this be removed?

Reply:
ah some copy paste shenanigans, cleaned that up in 5c3b851